src/chainstate/stacks/index/bits.rs

```diff
@@ -351,22 +352,27 @@ fn inner_read_nodetype_at_head<F: Read + Seek>(
         Error::CorruptionError(format!("read_node_type: Unknown trie node type {}", ptr_id))
     })? {
         TrieNodeID::Node4 => {
+            //eprintln!("Node4");
```

Comment:
Style nit: please remove commented-out code before merging. Thanks!

Reply:
Yeah there's a bit of tidying up to do before an eventual merge :)